<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="4" failed="4" ignored="0" total="8" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="All Test Suite" duration-ms="24702" started-at="2021-08-08T06:42:05Z" finished-at="2021-08-08T06:42:30Z">
    <groups>
      <group name="1">
        <method signature="Login.loginAsCampusUser()[pri:1, instance:test.java.uiAutomation.testcases.Login@29ba4338]" name="loginAsCampusUser" class="test.java.uiAutomation.testcases.Login"/>
        <method signature="Login.loginAsDistrictuser()[pri:0, instance:test.java.uiAutomation.testcases.Login@29ba4338]" name="loginAsDistrictuser" class="test.java.uiAutomation.testcases.Login"/>
        <method signature="Login.loginAsCampusUser()[pri:1, instance:test.java.uiAutomation.testcases.Login@3bbc39f8]" name="loginAsCampusUser" class="test.java.uiAutomation.testcases.Login"/>
        <method signature="Login.loginAsDistrictuser()[pri:0, instance:test.java.uiAutomation.testcases.Login@3bbc39f8]" name="loginAsDistrictuser" class="test.java.uiAutomation.testcases.Login"/>
      </group> <!-- 1 -->
    </groups>
    <test name="cucumberTest" duration-ms="8796" started-at="2021-08-08T06:42:05Z" finished-at="2021-08-08T06:42:14Z">
      <class name="test.java.uiAutomation.testcases.login2">
        <test-method status="PASS" signature="beforemethod()[pri:0, instance:test.java.uiAutomation.testcases.login2@57175e74]" name="beforemethod" is-config="true" duration-ms="1" started-at="2021-08-08T06:42:13Z" finished-at="2021-08-08T06:42:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method status="SKIP" signature="loginAsDistrictuser()[pri:0, instance:test.java.uiAutomation.testcases.login2@57175e74]" name="loginAsDistrictuser" duration-ms="0" depends-on-groups="1" started-at="2021-08-08T06:42:13Z" finished-at="2021-08-08T06:42:13Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method login2.loginAsDistrictuser()[pri:0, instance:test.java.uiAutomation.testcases.login2@57175e74] depends on not successfully finished methods in group "1"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method login2.loginAsDistrictuser()[pri:0, instance:test.java.uiAutomation.testcases.login2@57175e74] depends on not successfully finished methods in group "1"
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsDistrictuser -->
        <test-method status="SKIP" signature="loginAsCampusUser()[pri:1, instance:test.java.uiAutomation.testcases.login2@57175e74]" name="loginAsCampusUser" duration-ms="1" depends-on-groups="1" started-at="2021-08-08T06:42:13Z" finished-at="2021-08-08T06:42:13Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method login2.loginAsCampusUser()[pri:1, instance:test.java.uiAutomation.testcases.login2@57175e74] depends on not successfully finished methods in group "1"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method login2.loginAsCampusUser()[pri:1, instance:test.java.uiAutomation.testcases.login2@57175e74] depends on not successfully finished methods in group "1"
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsCampusUser -->
        <test-method status="PASS" signature="aftermethod()[pri:0, instance:test.java.uiAutomation.testcases.login2@57175e74]" name="aftermethod" is-config="true" duration-ms="0" started-at="2021-08-08T06:42:13Z" finished-at="2021-08-08T06:42:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftermethod -->
      </class> <!-- test.java.uiAutomation.testcases.login2 -->
      <class name="test.java.uiAutomation.testcases.Login">
        <test-method status="PASS" signature="beforeclass()[pri:0, instance:test.java.uiAutomation.testcases.Login@29ba4338]" name="beforeclass" is-config="true" duration-ms="0" started-at="2021-08-08T06:42:07Z" finished-at="2021-08-08T06:42:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeclass -->
        <test-method status="FAIL" signature="loginAsDistrictuser()[pri:0, instance:test.java.uiAutomation.testcases.Login@29ba4338]" name="loginAsDistrictuser" duration-ms="5657" started-at="2021-08-08T06:42:07Z" finished-at="2021-08-08T06:42:13Z">
          <exception class="org.openqa.selenium.NoSuchWindowException">
            <message>
              <![CDATA[no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=92.0.4515.131)
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'UW-IN-LPT0077', ip: '10.212.134.53', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_301'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 92.0.4515.131, chrome: {chromedriverVersion: 92.0.4515.43 (8c61b7e2989f2..., userDataDir: C:\Users\AMOHAM~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:49187}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: f21320d4416063ef5204746841341d58
*** Element info: {Using=xpath, value=//input[@id='login_username']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchWindowException: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=92.0.4515.131)
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'UW-IN-LPT0077', ip: '10.212.134.53', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_301'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 92.0.4515.131, chrome: {chromedriverVersion: 92.0.4515.43 (8c61b7e2989f2..., userDataDir: C:\Users\AMOHAM~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:49187}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: f21320d4416063ef5204746841341d58
*** Element info: {Using=xpath, value=//input[@id='login_username']}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at test.java.uiAutomation.core.BrowserFunctions.enterData(BrowserFunctions.java:101)
at test.java.uiAutomation.pagefactory.LpLoginPage.enterusername(LpLoginPage.java:26)
at test.java.uiAutomation.testcases.Login.loginAsDistrictuser(Login.java:17)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchWindowException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsDistrictuser -->
        <test-method status="PASS" signature="beforemethod()[pri:0, instance:test.java.uiAutomation.testcases.Login@29ba4338]" name="beforemethod" is-config="true" duration-ms="0" started-at="2021-08-08T06:42:07Z" finished-at="2021-08-08T06:42:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method status="PASS" signature="aftermethod()[pri:0, instance:test.java.uiAutomation.testcases.Login@29ba4338]" name="aftermethod" is-config="true" duration-ms="0" started-at="2021-08-08T06:42:13Z" finished-at="2021-08-08T06:42:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftermethod -->
        <test-method status="PASS" signature="beforemethod()[pri:0, instance:test.java.uiAutomation.testcases.Login@29ba4338]" name="beforemethod" is-config="true" duration-ms="0" started-at="2021-08-08T06:42:13Z" finished-at="2021-08-08T06:42:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method status="FAIL" signature="loginAsCampusUser()[pri:1, instance:test.java.uiAutomation.testcases.Login@29ba4338]" name="loginAsCampusUser" duration-ms="9" started-at="2021-08-08T06:42:13Z" finished-at="2021-08-08T06:42:13Z">
          <exception class="org.openqa.selenium.NoSuchWindowException">
            <message>
              <![CDATA[no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=92.0.4515.131)
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'UW-IN-LPT0077', ip: '10.212.134.53', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_301'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 92.0.4515.131, chrome: {chromedriverVersion: 92.0.4515.43 (8c61b7e2989f2..., userDataDir: C:\Users\AMOHAM~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:49187}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: f21320d4416063ef5204746841341d58
*** Element info: {Using=xpath, value=//input[@id='login_username']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchWindowException: no such window: target window already closed
from unknown error: web view not found
  (Session info: chrome=92.0.4515.131)
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'UW-IN-LPT0077', ip: '10.212.134.53', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_301'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 92.0.4515.131, chrome: {chromedriverVersion: 92.0.4515.43 (8c61b7e2989f2..., userDataDir: C:\Users\AMOHAM~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:49187}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: f21320d4416063ef5204746841341d58
*** Element info: {Using=xpath, value=//input[@id='login_username']}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at test.java.uiAutomation.core.BrowserFunctions.enterData(BrowserFunctions.java:101)
at test.java.uiAutomation.pagefactory.LpLoginPage.enterusername(LpLoginPage.java:26)
at test.java.uiAutomation.testcases.Login.loginAsCampusUser(Login.java:27)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchWindowException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsCampusUser -->
        <test-method status="PASS" signature="aftermethod()[pri:0, instance:test.java.uiAutomation.testcases.Login@29ba4338]" name="aftermethod" is-config="true" duration-ms="0" started-at="2021-08-08T06:42:13Z" finished-at="2021-08-08T06:42:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftermethod -->
        <test-method status="PASS" signature="afterclass()[pri:0, instance:test.java.uiAutomation.testcases.Login@29ba4338]" name="afterclass" is-config="true" duration-ms="0" started-at="2021-08-08T06:42:13Z" finished-at="2021-08-08T06:42:13Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterclass -->
      </class> <!-- test.java.uiAutomation.testcases.Login -->
      <class name="test.java.uiAutomation.testcases.suite">
        <test-method status="PASS" signature="beforetest(java.lang.String)[pri:0, instance:test.java.uiAutomation.testcases.suite@4206a205]" name="beforetest" is-config="true" duration-ms="2381" started-at="2021-08-08T06:42:05Z" finished-at="2021-08-08T06:42:07Z">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforetest -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:test.java.uiAutomation.testcases.suite@4206a205]" name="endTest" is-config="true" duration-ms="727" started-at="2021-08-08T06:42:13Z" finished-at="2021-08-08T06:42:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
      </class> <!-- test.java.uiAutomation.testcases.suite -->
    </test> <!-- cucumberTest -->
    <test name="cucumberTest1" duration-ms="15590" started-at="2021-08-08T06:42:14Z" finished-at="2021-08-08T06:42:30Z">
      <class name="test.java.uiAutomation.testcases.Login">
        <test-method status="PASS" signature="beforeclass()[pri:0, instance:test.java.uiAutomation.testcases.Login@3bbc39f8]" name="beforeclass" is-config="true" duration-ms="2" started-at="2021-08-08T06:42:15Z" finished-at="2021-08-08T06:42:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeclass -->
        <test-method status="PASS" signature="beforemethod()[pri:0, instance:test.java.uiAutomation.testcases.Login@3bbc39f8]" name="beforemethod" is-config="true" duration-ms="0" started-at="2021-08-08T06:42:15Z" finished-at="2021-08-08T06:42:15Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method status="FAIL" signature="loginAsDistrictuser()[pri:0, instance:test.java.uiAutomation.testcases.Login@3bbc39f8]" name="loginAsDistrictuser" duration-ms="1269" started-at="2021-08-08T06:42:15Z" finished-at="2021-08-08T06:42:17Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[unknown error: cannot determine loading status
from disconnected: received Inspector.detached event
  (Session info: chrome=92.0.4515.131)
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'UW-IN-LPT0077', ip: '10.212.134.53', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_301'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 92.0.4515.131, chrome: {chromedriverVersion: 92.0.4515.43 (8c61b7e2989f2..., userDataDir: C:\Users\AMOHAM~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:65036}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: edfa7e3167709ed01ae88f5b0987a535]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: unknown error: cannot determine loading status
from disconnected: received Inspector.detached event
  (Session info: chrome=92.0.4515.131)
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'UW-IN-LPT0077', ip: '10.212.134.53', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_301'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 92.0.4515.131, chrome: {chromedriverVersion: 92.0.4515.43 (8c61b7e2989f2..., userDataDir: C:\Users\AMOHAM~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:65036}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: edfa7e3167709ed01ae88f5b0987a535
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.get(RemoteWebDriver.java:277)
at test.java.uiAutomation.core.BrowserFunctions.goTo(BrowserFunctions.java:127)
at test.java.uiAutomation.pagefactory.LpLoginPage.gotoLp(LpLoginPage.java:36)
at test.java.uiAutomation.testcases.Login.loginAsDistrictuser(Login.java:16)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsDistrictuser -->
        <test-method status="PASS" signature="beforemethod()[pri:0, instance:test.java.uiAutomation.testcases.Login@3bbc39f8]" name="beforemethod" is-config="true" duration-ms="1" started-at="2021-08-08T06:42:17Z" finished-at="2021-08-08T06:42:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method status="PASS" signature="aftermethod()[pri:0, instance:test.java.uiAutomation.testcases.Login@3bbc39f8]" name="aftermethod" is-config="true" duration-ms="0" started-at="2021-08-08T06:42:17Z" finished-at="2021-08-08T06:42:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftermethod -->
        <test-method status="FAIL" signature="loginAsCampusUser()[pri:1, instance:test.java.uiAutomation.testcases.Login@3bbc39f8]" name="loginAsCampusUser" duration-ms="12359" started-at="2021-08-08T06:42:17Z" finished-at="2021-08-08T06:42:29Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[chrome not reachable
  (Session info: chrome=92.0.4515.131)
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'UW-IN-LPT0077', ip: '10.212.134.53', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_301'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 92.0.4515.131, chrome: {chromedriverVersion: 92.0.4515.43 (8c61b7e2989f2..., userDataDir: C:\Users\AMOHAM~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:65036}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: edfa7e3167709ed01ae88f5b0987a535
*** Element info: {Using=xpath, value=//input[@id='login_username']}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: chrome not reachable
  (Session info: chrome=92.0.4515.131)
Build info: version: 'unknown', revision: 'unknown', time: 'unknown'
System info: host: 'UW-IN-LPT0077', ip: '10.212.134.53', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_301'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 92.0.4515.131, chrome: {chromedriverVersion: 92.0.4515.43 (8c61b7e2989f2..., userDataDir: C:\Users\AMOHAM~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:65036}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: edfa7e3167709ed01ae88f5b0987a535
*** Element info: {Using=xpath, value=//input[@id='login_username']}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)
at java.lang.reflect.Constructor.newInstance(Unknown Source)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at test.java.uiAutomation.core.BrowserFunctions.enterData(BrowserFunctions.java:101)
at test.java.uiAutomation.pagefactory.LpLoginPage.enterusername(LpLoginPage.java:26)
at test.java.uiAutomation.testcases.Login.loginAsCampusUser(Login.java:27)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsCampusUser -->
        <test-method status="PASS" signature="aftermethod()[pri:0, instance:test.java.uiAutomation.testcases.Login@3bbc39f8]" name="aftermethod" is-config="true" duration-ms="2" started-at="2021-08-08T06:42:29Z" finished-at="2021-08-08T06:42:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftermethod -->
        <test-method status="PASS" signature="afterclass()[pri:0, instance:test.java.uiAutomation.testcases.Login@3bbc39f8]" name="afterclass" is-config="true" duration-ms="1" started-at="2021-08-08T06:42:29Z" finished-at="2021-08-08T06:42:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterclass -->
      </class> <!-- test.java.uiAutomation.testcases.Login -->
      <class name="test.java.uiAutomation.testcases.login2">
        <test-method status="PASS" signature="beforemethod()[pri:0, instance:test.java.uiAutomation.testcases.login2@4ae3c1cd]" name="beforemethod" is-config="true" duration-ms="2" started-at="2021-08-08T06:42:29Z" finished-at="2021-08-08T06:42:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforemethod -->
        <test-method status="SKIP" signature="loginAsDistrictuser()[pri:0, instance:test.java.uiAutomation.testcases.login2@4ae3c1cd]" name="loginAsDistrictuser" duration-ms="0" depends-on-groups="1" started-at="2021-08-08T06:42:29Z" finished-at="2021-08-08T06:42:29Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method login2.loginAsDistrictuser()[pri:0, instance:test.java.uiAutomation.testcases.login2@4ae3c1cd] depends on not successfully finished methods in group "1"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method login2.loginAsDistrictuser()[pri:0, instance:test.java.uiAutomation.testcases.login2@4ae3c1cd] depends on not successfully finished methods in group "1"
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsDistrictuser -->
        <test-method status="SKIP" signature="loginAsCampusUser()[pri:1, instance:test.java.uiAutomation.testcases.login2@4ae3c1cd]" name="loginAsCampusUser" duration-ms="0" depends-on-groups="1" started-at="2021-08-08T06:42:29Z" finished-at="2021-08-08T06:42:29Z">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method login2.loginAsCampusUser()[pri:1, instance:test.java.uiAutomation.testcases.login2@4ae3c1cd] depends on not successfully finished methods in group "1"]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method login2.loginAsCampusUser()[pri:1, instance:test.java.uiAutomation.testcases.login2@4ae3c1cd] depends on not successfully finished methods in group "1"
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:888)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.TestNG.privateMain(TestNG.java:1354)
at org.testng.TestNG.main(TestNG.java:1323)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginAsCampusUser -->
        <test-method status="PASS" signature="aftermethod()[pri:0, instance:test.java.uiAutomation.testcases.login2@4ae3c1cd]" name="aftermethod" is-config="true" duration-ms="2" started-at="2021-08-08T06:42:29Z" finished-at="2021-08-08T06:42:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- aftermethod -->
      </class> <!-- test.java.uiAutomation.testcases.login2 -->
      <class name="test.java.uiAutomation.testcases.suite">
        <test-method status="PASS" signature="beforetest(java.lang.String)[pri:0, instance:test.java.uiAutomation.testcases.suite@5ea434c8]" name="beforetest" is-config="true" duration-ms="1248" started-at="2021-08-08T06:42:14Z" finished-at="2021-08-08T06:42:15Z">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforetest -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:test.java.uiAutomation.testcases.suite@5ea434c8]" name="endTest" is-config="true" duration-ms="694" started-at="2021-08-08T06:42:29Z" finished-at="2021-08-08T06:42:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
      </class> <!-- test.java.uiAutomation.testcases.suite -->
    </test> <!-- cucumberTest1 -->
  </suite> <!-- All Test Suite -->
</testng-results>
